centreon-engine/src/host.cc

```diff
@@ -3521,8 +3513,10 @@ int host::process_check_result_3x(enum host::host_state new_state,
   if (reschedule_check) {
     engine_logger(dbg_checks, more)
         << "Rescheduling next check of host at " << my_ctime(&next_check);
-    log_v2::checks()->debug("Rescheduling next check of host at {}",
-                            my_ctime(&next_check));
+    log_v2::checks()->debug(
+        "Rescheduling next check of host: {} of last check at {} and next "
+        "check at {}",
+        _name, get_last_check(), next_check);
```

Comment:
In the previous log there was a my_ctime call. And I think spdlog proposes a special function to write dates correctly instead of timestamp.